CHANGELIST.md

```diff
@@ -0,0 +1,8 @@
+## V 1.2
```

Comment:
Should the first release be 1.0 rather than 1.2. Maven pacakge name should sync with that ... i think that's 1.0.0.

Reply:
Fixed